src/features/Signup/SignupPage.jsx

```diff
@@ -73,7 +73,10 @@ const SignupPage = (props) => {
       },
       onError: (error) => {
         if (error.response.data && error.response.data.message) {
-          const errorMessage = error.response.data.message;
+          let errorMessage = error.response.data.message;
+          if(errorMessage === "tenant already exists"){
+            errorMessage = "Something went wrong. Please check if you've already signed up";
```

Comment:
Can we use this message instead? Also lets ensure that this message change happens on server.

If the email address is not already in use, you'll receive a link to complete your signup process.

Use the same message for both success and failure.